tests/ClientTest.php

```diff
@@ -24,7 +24,60 @@ public function testLogin()
         $clientMock->login();
 
-        $this->assertEquals('TEST_TOKEN', $clientMock->getToken());
+        $this->assertEquals('TEST_TOKEN', $clientMock->getAccessToken());
+    }
+
+    public function testRefreshLogin()
+    {
+        $clientMock = $this->getClientMock(array('call', 'getEndpoint'));
+
+        $body = '{"accessToken":"TEST_TOKEN","tokenType":"Bearer","expiresIn":86400,"refreshToken":"REFERSH_TOKEN"}';
+        $responseMock = $this->getResponseMock($body, 200);
+
+        $clientMock->expects($this->once())->method('getEndpoint')->with('login/refresh')->willReturn('http://testing.com/rest/login/refresh');
+        $clientMock->expects($this->once())->method('call')->with('POST', 'http://testing.com/rest/login/refresh')->will($this->returnValue($responseMock));
+        $clientMock->refreshLogin();
+
+        $this->assertEquals('TEST_TOKEN', $clientMock->getAccessToken());
+        $this->assertEquals('REFERSH_TOKEN', $clientMock->getRefreshToken());
+    }
+
+    public function testTokensAreRefreshed()
+    {
+        $webstoresUnauthorizedRequestMock = $this->getRequestMock(['send']);
+        $refreshRequestMock = $this->getRequestMock(['send']);
+        $webstoresAuthorizedRequestMock = $this->getRequestMock(['send']);
+
+        $clientMock = $this->getClientMock(['createRequest', 'getUrl']);
+
+        $refreshBody = '{"accessToken":"TEST_TOKEN","tokenType":"Bearer","expiresIn":86400,"refreshToken":"REFERSH_TOKEN"}';
+        $refreshResponse = $this->getResponseMock($refreshBody, 200);
+        $refreshRequestMock->expects($this->once())->method('send')->willReturn($refreshResponse);
+
+        $webstoresUnauthorizedResponse = $this->getResponseMock('Failed', 401);
+        $webstoresUnauthorizedResponse->expects($this->any())->method('getReasonPhrase')->willReturn('Unauthorized');
+        $webstoresUnauthorizedRequestMock->expects($this->once())->method('send')->willReturn($webstoresUnauthorizedResponse);
+
+        $webstoresAuthorizedResponse = $this->getResponseMock('{"test": "success"}', 200);
+        $webstoresAuthorizedRequestMock->expects($this->once())->method('send')->willReturn($webstoresAuthorizedResponse);
+
+        $clientMock->expects($this->exactly(2))->method('getUrl')->willReturn('test.com/');
+
+        $clientMock->method('createRequest')
+            ->withConsecutive(
+                ['GET', 'https://test.com/webstores', null],
+                ['POST', 'https://test.com/login/refresh', ['refresh_token' => null]],
+                ['GET', 'https://test.com/webstores', null]
+            )->willReturnOnConsecutiveCalls(
+                $webstoresUnauthorizedRequestMock,
+                $refreshRequestMock,
+                $webstoresAuthorizedRequestMock
+            );
+
+        $clientMock->getWebstores();
+
+        $this->assertEquals($clientMock->getAccessToken(), "TEST_TOKEN");
+        $this->assertEquals($clientMock->getRefreshToken(), "REFERSH_TOKEN");
```

Comment:
* Flip the order of these parameters since the first one is supposed to be the expected value.
* Use single quotes.